packages/lib/services/spellChecker/SpellCheckerService.ts

```diff
@@ -35,23 +35,30 @@ export default class SpellCheckerService {
 	}
 
 	private async addLatestSelectedLanguage(language: string) {
+		// This function will add selected languages to the history. History size will be capped at LANGUAGES_HISTORY_SIZE_MAX,
+		// but it can be bigger. Enabled languages will always be in the history, even if it count greater then
+		// LANGUAGES_HISTORY_SIZE_MAX, in such case if one of the languages will be disabled it will disappear from history.
+		const LANGUAGES_HISTORY_SIZE_MAX = 5;
```

Comment:
It looks like [the codebase should be using camelCase for constants](https://github.com/laurent22/joplin/pull/6586#discussion_r911502889). (I did suggest SNAKE_CASE originally, but I was wrong! :) )

Reply:
Made the change.